frontend/src/App/App.jsx

```diff
@@ -75,7 +75,12 @@ class AppComponent extends React.Component {
       'confirm',
       'confirm-invite',
     ];
-    return existedPaths.find((path) => window.location.pathname.includes(path));
+
+    const subpath = getSubpath();
+    const subpathArray = subpath ? subpath.split('/').filter((path) => path != '') : [];
+    const pathnames = window.location.pathname.split('/')?.filter((path) => path != '');
```

Comment:
Do you think it'd be simple to reason about if we get the subpath and then split it out from `window.location.pathname` and go about the normal logic?

Reply:
can keep for refactoring later